### PR TITLE
Add EIA ID to Utility

### PR DESCRIPTION
**Pull request by macintoshpie**
## Motivation
It would be nice to have these IDs for identifying utility types: https://openei.org/wiki/List_of_United_States_Utility_Companies_and_Aliases#cite_note-1

### CODE REVIEW

**Comment by macintoshpie** [comment]
@corymosiman12 Can you improve the description if needed?

**Review by markborkum — COMMENTED** [review]
In the linked webpage, the IDs appear to be `xs:nonNegativeInteger`, not `xs:string`, e.g.,

> 3 Phases Energy Services (ID # 21093)

File: BuildingSync.xsd [hunk 1]
```diff
@@ -3736,6 +3736,11 @@
           </xs:restriction>
         </xs:simpleType>
       </xs:element>
+      <xs:element name="EIAUtilityID" type="xs:nonNegativeInteger" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>ID of the OpenEI EIA Utility.</xs:documentation>
```
Comment (corymosiman12) [inline BuildingSync.xsd:3741]: "EIA Utility ID as found in EIA-861 and as available in OpenEI"

**Review by corymosiman12 — CHANGES_REQUESTED** [review]

**Review by corymosiman12 — APPROVED** [review]

**Review by markborkum — APPROVED** [review]
Changes look good, thanks.

**Pull request merged**